mrcnn/model.py

```diff
@@ -8,33 +8,34 @@
 """
 
 import os
-import random
 import datetime
 import re
 import math
-import logging
 from collections import OrderedDict
 import multiprocessing
 import numpy as np
 import tensorflow as tf
-import keras
-import keras.backend as K
-import keras.layers as KL
-import keras.engine as KE
-import keras.models as KM
+import tensorflow.keras as keras
+import tensorflow.keras.backend as K
+import tensorflow.keras.layers as KL
+import tensorflow.keras.layers as KE
```

Comment:
This line should probably be removed, since there are only a few places that reference KE and and they can all be updated to KL